src/tests/Microsoft.PowerFx.Core.Tests/CheckResultTests.cs

```diff
@@ -0,0 +1,277 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerFx.Syntax;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class CheckResultTests
+    {
+        [Fact]
+        public void Ctors()
+        {
+            // CheckResult must have an engine so that it can invoke operations.
+            Assert.Throws<ArgumentNullException>(() => new CheckResult((Engine)null));
```

Comment:
>Throws [](http://example.com/codeflow?start=19&length=6)

Can we check the message as well? (applies to all Assert.Throws)

Reply:
our other Throws test don't do this.